.github/workflows/build_release.yml

```
@@ -0,0 +1,74 @@
+            - main
+
+permissions:
+    contents: write
+
+jobs:
+    build:
+        name: Build & Release
+        runs-on: ubuntu-latest # we can select macos-latest to be able to get ios build
+        steps:
+            - name: Checkout Repository
+              uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)